dev/tasks/build-images

```diff
@@ -19,6 +19,17 @@ set -o pipefail
 
 REPO_ROOT="$(git rev-parse --show-toplevel)"
 
+# Force the image names, to avoid depending on the current gcloud project
```

Comment:
if we accept this we probably can remove my hack to inject the `mockProject` id to the GH action steps 😛